lib/vagrant-libvirt/action.rb

```diff
@@ -128,12 +128,6 @@ def self.action_up
       private_class_method def self.action_start
         Vagrant::Action::Builder.new.tap do |b|
           b.use Call, IsRunning do |env, b2|
-            # If the VM is running, run the necessary provisioners
-            if env[:result]
-              b2.use action_provision
-              next
```

Comment:
I hadn't actually checked whether `next` could be removed as well.

Reply:
yea, just removing `b2.use action_provision` and leaving the rest of the block still solved the issue when testing. I just readded it back unless you feel otherwise.